src/Microsoft.IdentityModel.Protocols.WsFederation/LogMessages.cs

```diff
@@ -40,6 +40,7 @@ internal static class LogMessages
         internal const string IDX10900 = "IDX10900: Building wsfederation message from query string: '{0}'.";
         internal const string IDX10901 = "IDX10901: Building wsfederation message from uri: '{0}'.";
         internal const string IDX10902 = "IDX10902: Token is not found in wresult";
+        internal const string IDX10903 = "IDX10903: More than one token are found in wresult";
```

Comment:
Multiple tokens were found in the RequestSecurityTokenCollection. Only a single token is supported.